src/pages/Validators/Stats/ActiveValidators.tsx

```diff
@@ -1,37 +1,26 @@
 // Copyright 2022 @paritytech/polkadot-staking-dashboard authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import BN from 'bn.js';
 import { useStaking } from 'contexts/Staking';
 import { Pie } from 'library/StatBoxList/Pie';
 import { useTranslation } from 'react-i18next';
-import { toFixedIfNecessary } from 'Utils';
 
 const ActiveValidatorsStatBox = () => {
-  const { staking, eraStakers } = useStaking();
-  const { validatorCount } = staking;
+  const { eraStakers } = useStaking();
   const { activeValidators } = eraStakers;
   const { t } = useTranslation('pages');
 
-  // active validators as percent
-  let activeValidatorsAsPercent = 0;
-  if (validatorCount.gt(new BN(0))) {
-    activeValidatorsAsPercent =
-      activeValidators / (validatorCount.toNumber() * 0.01);
-  }
-
   const params = {
     label: t('validators.active_validators'),
     stat: {
       value: activeValidators,
-      total: validatorCount.toNumber(),
       unit: '',
     },
     graph: {
       value1: activeValidators,
-      value2: validatorCount.sub(new BN(activeValidators)).toNumber(),
+      value2: 0,
```

Comment:
Do I understand this correctly - this is because in Aleph Zero network there's no such thing as "max validators count"?

Reply:
Yup, it's shown in the box on the left.
<img width="593" alt="image" src="https://user-images.githubusercontent.com/7670051/205683408-11144050-69eb-4739-8d3c-bf559e5a6c19.png">